<launch>
	<arg name="ip" default="127.0.0.1"/>
	<arg name="port" default="5005"/>
	<node pkg="bitbots_hardware_rqt" type="cpu_info_publisher.py" name="cpu_info" output="screen">
		<param name="cpu_name" value="cpu_name"/>
	</node>
	<node pkg="bitbots_hardware_rqt" type="cpu_info_publisher.py" name="cpu_info2" output="screen">
			<param name="cpu_name" value="cpu_name2"/>
	</node>
	<!-- <node pkg="bitbots_hardware_rqt" type="battery_info_publisher.py" name="battery_info" output="screen"/> -->
	<node pkg="bitbots_hardware_rqt" type="hardware_tool.py" name="hardware_tool" output="screen">
		<rosparam command="load" file="$(find bitbots_hardware_rqt)/config/hardware_rqt_settings.yaml"/>
		<param name="rqt_ip" value="$(arg ip)"/>
		<param name="hardware_rqt_port" value="$(arg port)"/>
	</node>
</launch>
